docs/getting_started.md

```diff
@@ -7,10 +7,20 @@ This image, designed for quick local testing, launches the Jaeger UI, collector,
 The simplest way to start the all in one docker image is to use the pre-built image published to DockerHub.
 
 ```bash
-docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 -p16686:16686 jaegertracing/all-in-one:latest
+docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 -p16686:16686 -p14268:14268 jaegertracing/all-in-one:latest
 ```
 
-You can then navigate to `http://localhost:16686` to access the Jaeger UI. 
+You can then navigate to `http://localhost:16686` to access the Jaeger UI.
+
+Port | Protocol | Component | Function
```

Comment:
Agree, also the docker command itself isn't readable in it's current state. Perhaps we can refactor the deployment page, and link to that instead. I'll give it some thought tonight/tomorrow.